### ISSUE TITLE
@ApiParam value is not respected

### ISSUE THREAD

**Issue by sta-szek** [springfox/springfox#1597]
my code is annotation on parameter `@ApiParam(value = "example")` .
Swagger shows:
![image](https://cloud.githubusercontent.com/assets/6240704/20919989/9aedbe6c-bb9e-11e6-8fe8-9b037c2406c4.png)
so the second `sort` should be `example` but is not.

Im using springfox swagger
compile("io.springfox:springfox-swagger2:2.6.1")
compile("io.springfox:springfox-swagger-ui:2.6.1")

With version 2.5.1 everything was working. Nothing was changed in configuration, just version upgrade.

**Comment by dilipkrish** [comment]
Could you share what you're endpoint method signature looks like?

**Comment by sta-szek** [comment]
```
public Object getSmth(
@ApiParam(value = "example", name = "sort")
@RequestParam(name = "sort", required = false, defaultValue = "") final Sort sort) {}

public class Sort {
    private String fieldName;
    private Direction direction = Direction.ASC;
    // getters, setters, constructors,....
}
```

**Comment by dilipkrish** [comment]
This appears to be fixed in (yet to be released) 2.7.0
